ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -328,6 +328,35 @@ class FamilyReads extends React.PureComponent {
     )
   }
 
+  getSampleColors = () => {
+    const { openFamily } = this.state
+    const { igvSamplesByFamilySampleIndividual, individualsByGuid } = this.props
+    const igvSampleIndividuals = (
+      openFamily && (igvSamplesByFamilySampleIndividual || {})[openFamily]) || {}
+    return Object.keys(igvSampleIndividuals[GCNV_TYPE] || {}).reduce(
+      (acc, iGuid) => ({
+        ...acc,
+        [iGuid]: individualsByGuid[iGuid].affected === AFFECTED ? 'red' : 'blue',
+      }), {},
+    )
+  }
+
+  getSampleColorPanel = () => {
+    const { individualsByGuid } = this.props
+    const sampleColors = this.getSampleColors()
+    return Object.entries(sampleColors).map(
+      ([iGuid, sampleColor]) => (
+        <div key={iGuid}>
+          <ColoredSpan color="black">
+            {individualsByGuid[iGuid].displayName}
+            &nbsp;
+          </ColoredSpan>
+          <ColoredSpan backgroundColor={sampleColor}>&nbsp;</ColoredSpan>
```

Comment:
The name text should be the same color as everything else in this side bar. If thats black, fine, if thats grey, fine, but you should not be adding special color handling for it. 

For the icon itself, use an actual icon, not a span with a background color. Its always best practice for the HTML to be as semantically close to what it is doing as possible- a span is usually an inline container for content while an icon is actually an icon. Given that the only colors we support are red and blue which are standard colors handled by semantic ui this doesn't even need to be a styled component, you can just do `<Icon name="square full" color={sampleColor} />`

Reply:
The name text's default color is grey. But the color is black for the other text (e.g. the track names) in the sidebar. Should I keep the sample name text in grey?

Reply:
Without the special styling, the sidebar looks as below figure.
![image](https://user-images.githubusercontent.com/33550059/142653582-f6e81b0c-3ba8-4a70-b509-323dfb278274.png)

Reply:
probably you should  put the text it inside a `label`, or just in general use the same way of styling this as everything else on the sidebar which probably involves wrapping it in something. Since nothing else here is using ColoredSpan this shouldn't have to either

Also, I know its not in the original screenshot, but can we put the colors to the left of the labels instead of the right so the alignment is nicer? Also, we should add a divider for this section as well, with the title "gCNV Samples"

Reply:
I can add the divider.

For the colored icon, maybe you want it to be consistent with the TGG-Viewer. It is on the right for TGG-viewer.

Reply:
Given that I explicitly said "I know its not in the original screenshot" I clearly DON'T want it to look like the TGG viewer, which is a totally different and unrelated application to seqr that we have literally no need or desire to conform to. I want it to actually look good, which is why I suggested what I did